lib/cli/src/generators/MITHRIL/index.js

```diff
@@ -35,6 +42,7 @@ export default async (npmOptions, { storyFormat = 'csf' }) => {
     `@storybook/mithril@${storybookVersion}`,
     `@storybook/addon-actions@${actionsVersion}`,
     `@storybook/addon-links@${linksVersion}`,
+    `@storybook/addon-knobs@${knobsVersion}`,
```

Comment:
I added `knobs` because it is listed in `addons` of the configuration: https://github.com/storybookjs/storybook/blob/3dd714d3532687d958c41e093187b84ffca603ee/lib/cli/src/generators/MITHRIL/template-csf/.storybook/main.js but it is not used in template stories so maybe it should be deleted from configuration instead? 🤷‍♂ 

At the opposite `@storybook/addon-links` is added and used in template stories but is not declared in `addons` config, so we can maybe remove it?